.circleci/config.yml

```diff
@@ -549,18 +552,19 @@ jobs:
           name: Update Formula
           command: |
             source .circleci/get_archive_tag.sh
-            UPDATED_SHA256=$(openssl dgst -sha256 dist/archive/raiden-${ARCHIVE_TAG}-macOS.zip)
-            FORMULA_FILE="homebrew-raiden/raiden.rb"
-            sed -i .bak -E "s/[0-9]+\.[0-9]+\.[0-9]+/${ARCHIVE_TAG/v/}/g" ${FORMULA_FILE}
-            sed -i .bak -E "s/sha256 \"[a-f0-9]\{64\}\"/sha256 \"${UPDATED_SHA256: -64}\"/g" ${FORMULA_FILE}
-            rm $FORMULA_FILE.bak
+            export UPDATED_SHA256=$(openssl dgst -sha256 dist/archive/raiden-${ARCHIVE_TAG}-macOS-x86_64.zip)
+            export FORMULA_FILE="homebrew-raiden/raiden.rb"
+            sed -i.bak -r "s/[0-9]+\.[0-9]+\.[0-9]+(-?rc.)?/${ARCHIVE_TAG/v/}/g" ${FORMULA_FILE}
```

Comment:
Was removing the space in `-i.bak` done on purpose?

Reply:
Yes it was. Previously (on Travis) this ran on macOS which has BSD sed. On Circle this is now running in a linux container with gnu sed which (for whatever reason) doesn't accept a space after the `-i` option.